web/src/components/TaskCtaArea.tsx

```diff
@@ -0,0 +1,123 @@
+import { PrimaryButtonDropdown } from "@/components/njwds-extended/PrimaryButtonDropdown";
+import { TaskCTA } from "@/components/TaskCTA";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { PostOnboarding, Task } from "@/lib/types/types";
+import { openInNewTab } from "@/lib/utils/helpers";
+import { ProfileData } from "@businessnjgovnavigator/shared/index";
+import { useRouter } from "next/router";
+import React, { ReactElement } from "react";
+
+interface ButtonsArrayElement {
+  text: string;
+  destination: string;
+  onClick: () => void;
+}
+
+const TaskFooter = (props: React.ComponentPropsWithoutRef<"div">): JSX.Element => {
+  const { children, ...attributes } = props;
+  return (
+    <div
+      data-testid="cta-area"
+      style={{ marginTop: "2rem" }}
+      className="flex flex-justify-end bg-base-lightest margin-x-neg-4 padding-3 margin-bottom-neg-4 flex-column mobile-lg:flex-row radius-bottom-lg"
+      {...attributes}
+    >
+      <div className="mobile-lg:margin-bottom-0 margin-bottom-1">{children}</div>
+    </div>
+  );
+};
+
+interface Props {
+  postOnboardingQuestion: PostOnboarding | undefined;
+  task?: Task | undefined;
+  forceDisplay?: boolean | undefined;
```

Comment:
[pebble] What's the purpose of this prop? If I'm reading correctly, it seems this is always `true` in the app? Which is an indicator that it is unnecessary and should be removed. 

If it actually is necessary behavior, I'd expect it to be covered in the unit tests for this component.

Reply:
I won't comment all over the place for the same question, but I'm also unsure about why the if/else openInNewTab controlled by this prop?

Reply:
`CMS_ONLY_forceDisplay` is good, that helps. For what it's worth, I don't think it's necessary at all though. Nothing in the CMS preview panel is clickable or interactable and there's no reason this one needs to be.

Reply:
Oh, I made it clickable so users can see the dropdown list, oops. Should I make it unclickable again?

Reply:
🤷 

- it's already built
- but it adds a non-insignificant amount of complexity to the code
- but it's useful for the CMS team 
- but it's not something we do (or should plan to do) in general in the CMS, imo

Your call